impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -194,35 +243,24 @@ bool BufferBindingsGLES::BindUniformBuffer(const ProcTableGLES& gl,
     return false;
   }
 
-  for (const auto& member : metadata->members) {
-    if (member.type == ShaderType::kVoid) {
-      // Void types are used for padding. We are obviously not going to find
-      // mappings for these. Keep going.
+  const auto& locations = ComputeUniformLocations(metadata);
+  for (auto i = 0u; i < metadata->members.size(); i++) {
+    const auto& member = metadata->members[i];
+    auto location = locations[i];
+    if (location == -1) {
```

Comment:
Preserve the comment about inactive uniformed being optimized out?

Reply:
Oh, I see it up top. Maybe just "void type or inactive".

Reply:
Done